grafana/dashboards/overview.json

```diff
@@ -2169,4 +2169,4 @@
   "uid": "kOuP_Fggz",
   "version": 9,
```

Comment:
version should be updated

Reply:
@JakobLichterfeld  thanks, I missed it, new commit is ready